views/help/partials/login_failed_hint.hbs

```diff
@@ -0,0 +1,36 @@
+
+<div class="login-failed-container">
+	{{#if notification.statusCode}}
+		<h4>{{$t "login.headline.LoginFailedHint"}}</h4>
+			<div class="steps">
+				<li> {{$t "login.text.LoginFailedFirstHint"}}</li>
+				<p>{{ $t "login.text.LoginFailedFirstHintText"}}</p>
+					<img class="browser" src="/images/chrome.png" alt="Firefox" height="42" width="42">
+					<img class="browser" src="/images/firefox.png" alt="Chrome" height="42" width="42">
+					<img class="browser" src="/images/safari.png" alt="Safari" height="42" width="42">
+			</div>
+			<div class="steps">
+				<li> {{$t "login.text.LoginFailedSecondHint"}}</li>
+				<a href="https://support.google.com/accounts/answer/32050?co=GENIE.Platform%3DDesktop&hl=de" target="_blank">Hier findest du heraus wie du deine Cookies löschst.</a>
+			</div>
+			<div class="steps">
+				<li>{{$t "login.text.LoginFailedThirdHint"}}</li>
+			</div>
+			<div class="steps">
+				<li> {{$t "login.text.LoginFailedFourthHint"}}</li>
+				<p>{{ $t "login.text.LoginFailedFourthHintText"}}</p>
+				<img class="screenshot"src="/images/login-fail.png" alt="Login failed">
+			</div>
+			<div class="steps">
+				<li>{{$t "login.text.LoginFailedFithHint"}}</li>
+				<p>{{ $t "login.text.LoginFailedFithHintText"}}</p>
+				<div class="logo-container">
+					<a href="https://schul-cloud.org/"><img class="sc-logo" src="/images/schulcoud-logo.png" alt="Logo Schul-Cloud"target="_blank" width="200"></a>
+					<a href="https://brandenburg.schul-cloud.org"><img class="sc-logo" src="/images/brb-logo.png" alt="Logo brb"target="_blank" width="200"></a>
+					<a href="https://schulcloud-thueringen.de"><img class="sc-logo" src="/images/thr-logo.png" alt="Logo thr"target="_blank" width="200"></a>
+					<a href="https://niedersachsen.cloud/"><img class="sc-logo" src="/images/n21-logo.png" alt="Logo n21"target="_blank" width="200"></a>
+					<a href="https://open.schul-cloud.org/"><img class="sc-logo" src="/images/open-logo.png" alt="Logo open"target="_blank" width="200"></a>
```

Comment:
same for the other img_alt

Reply:
![Opera Momentaufnahme_2020-05-11_142153_localhost](https://user-images.githubusercontent.com/36112357/81561021-cce09500-9392-11ea-8a58-5176635348c5.png)
Maybe we can place 3 logos side by side on desktop?

Reply:
@sarahschaefer what would you prefer?